models/schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+models:    
+  - name: mp_events
+    description: "This table collects all media player events as a base table"
+    columns:
+      - name: event_id
+        description: "The primary key of this table"
+        tests:
+          - unique
+          - not_null
+  - name: interactions
+    description: "This table shows all media player events and calculates respective time periods"
+    columns:
+      - name: event_id
+        description: "The primary key of this table"
+        tests:
+          - unique
+          - not_null
+  - name: plays_by_pageview
+    description: "This table aggregates media player interactions to a pageview level"
+    columns:
+      - name: play_id
+        description: "The primary key of this table that is unique for each media_id and pageview"
```

Comment:
I think when referencing tables or other columns it makes sense to put them around single quotes for markdown purposes, e.g. for each `media_id` and `page_view_id`